projects/packages/assets/tests/php/test-assets.php

```diff
@@ -729,7 +729,7 @@ function ( $v ) {
 
 		// @phan-suppress-next-line PhanDeprecatedFunction -- Keep using setMethods until we drop PHP 7.0 support.
 		$mock = $this->getMockBuilder( \stdClass::class )
-			->setMethods( array( 'add', 'add_inline_script' ) )
+			->setMethods( array( 'add', 'add_inline_script', 'add_data' ) )
```

Comment:
Should we add some `$mock->expects( ... )->method( 'add_data' )` to check that the proper calls were made for that too?

Reply:
Sure thing, good idea. Done in 1503ba497019f68c2b381b0d2fb2341dfb2326bf